src/System.Private.CoreLib/shared/System/Globalization/CultureInfo.cs

```diff
@@ -175,13 +175,14 @@ public CultureInfo(string name, bool useUserOverride)
             }
 
             // Get our data providing record
-            _cultureData = CultureData.GetCultureData(name, useUserOverride);
+            CultureData? cultureData = CultureData.GetCultureData(name, useUserOverride);
 
-            if (_cultureData == null)
+            if (cultureData == null)
             {
                 throw new CultureNotFoundException(nameof(name), name, SR.Argument_CultureNotSupported);
             }
 
+            _cultureData = cultureData;
```

Comment:
Will this affect IL? Is it worth just to avoid making `_cultureData` non-nullable?

Reply:
it is nullable - probably will affect it a bit - but IMO worth it vs 25 `!`s